client/my-sites/site-settings/form-general.jsx

```diff
@@ -260,12 +290,12 @@ const FormGeneral = React.createClass( {
 						name="blog_public"
 						value="1"
 						checked={ 1 === parseInt( this.state.blog_public, 10 ) }
-						onChange={ this.handleRadio }
+						onChange={ this.handleChange }
 						disabled={ this.state.fetchingSettings }
 						onClick={ this.onRecordEvent( 'Clicked Site Visibility Radio Button' ) } />
-					<span>{ this.translate( 'Public' ) }</span>
+					<span>{ translate( 'Public' ) }</span>
 					<FormSettingExplanation isIndented>
```

Comment:
Should this be indented? @MichaelArestad the grey text under each option

![screen shot 2016-12-21 at 11 30 14 am](https://cloud.githubusercontent.com/assets/541093/21397220/ee1c6916-c770-11e6-843b-ca26e090360b.png)